source/deployment/incident-response-and-monitoring.rst

```diff
@@ -0,0 +1,47 @@
+Incident Response and Monitoring Tools
+============================================
+
+Our `Integrations Directory <https://integrations.mattermost.com>`_ has several integrations to connect incident response and monitoring tools in Mattermost.
+
+Here are some popular options below, including self-hosted on-prem or self-hosted private cloud solutions, and vendor-hosted SaaS solutions.
+
+AWS CloudWatch SNS
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Two-way integration between Mattermost and Amazon AWS SNS, developed by Carlos Tadeu Panato Junior, supported by Mattermost.
+ - Receive SNS notifications from Alerts created by Amazon AWS CloudWatch and sent via AWS SNS.
+ - Get operational data and actionable insights to monitor Mattermost in order to respond to system-wide performance changes.
```

Comment:
Is it 'performance changes'?

Reply:
Changed text to become "performance changes system-wide"